cirq-google/cirq_google/experimental/noise_models/calibration_to_noise_properties.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Tuple, Type
+import numpy as np
+
+import cirq, cirq_google
```

Comment:
Nit: circular ?

Reply:
Turns out there's already a circular import in `google_noise_properties`:
https://github.com/quantumlib/Cirq/blob/e31b48d04c7a3dea6bc2e5e320b0c606bbf13700/cirq-google/cirq_google/devices/google_noise_properties.py#L22

More specifically, it uses `cirq_google.ops` (which comes after `cg.devices` in the hierarchy) for `SycamoreGate`. So I have some untangling to do. Possible side effects:
- Might remove public `*_QUBIT_GATES` sets in favor of in-method sets
- Probably moving calibration-to-noise out of experimental
- _Might_ have to move `google_noise_properties` as well (unlikely)